src/frontend/context/Valutakurs/ValutakursSkjemaContext.tsx

```diff
@@ -137,20 +133,60 @@ const useValutakursSkjema = ({ barnIValutakurs, valutakurs }: IProps) => {
         skjemanavn: valutakursFeilmeldingId(valutakurs),
     });
 
+    const [tidligereValutakurs, settTidligereValutakurs] = useState<IRestValutakurs>();
+
+    const settDatofelterTilDefault = () => {
+        const nyValutakursdato = valutakurs.valutakursdato
+            ? new Date(valutakurs.valutakursdato)
+            : undefined;
+        skjema.felter.valutakursdato.validerOgSettFelt(nyValutakursdato);
+    };
+
+    const tilbakestillFelterTilDefault = () => {
+        skjema.felter.periodeId.nullstill();
+        skjema.felter.id.nullstill();
+        skjema.felter.status.nullstill();
+        skjema.felter.initielFom.nullstill();
+        skjema.felter.barnIdenter.nullstill();
+        skjema.felter.periode.nullstill();
+        skjema.felter.valutakode.nullstill();
+        skjema.felter.kurs.nullstill();
```

Comment:
Skulle gjerne heller har brukt nullstillSkjema() men da ender vi opp med feil state for datofeltet, som først blir nullstilt og deretter satt til default-verdi. Skulle trodd feltet da endte opp med default-verdien, men ser ut som den ender opp med undefined (må være noe lag ett sted i nullstillingen? 🤷 )

Reply:
Litt usikker på hva som menes her. Ser ut som deafult-verdien settes til undefined, og at det da gir mening at den blir undefined når den nullstilles til defaultveriden? 🤔